swagger.yaml

```diff
@@ -466,6 +472,33 @@ paths:
           description: "Version not found"
         500:
           $ref: '#/responses/InternalError'
+  /datasets/{id}/editions/{edition}/versions/{version}/observation:
+    get:
+      tags:
+      - "Public"
+      summary: "Get a specific observation from version"
+      description: "Get observation from a version of the dataset by providing a single dimension option for each dimension"
+      parameters:
+        - $ref: '#/parameters/edition'
+        - $ref: '#/parameters/id'
+        - $ref: '#/parameters/version'
+        - $ref: '#/parameters/dimension_options'
+      responses:
+        200:
+          description: "Json object containing all metadata for a version"
+          schema:
+            $ref: '#/definitions/ObservationEndpoint'
+        400:
+          description: "Invalid request"
+            
+        404:
+          description: |
+            Resource not found, reasons can be one of the following:
+              * dataset id was incorrect
+              * edition was incorrect
+              * verion was incorrect
```

Comment:
Incorrect dimensions are fine, as long as the combination of valid dimensions specified provide 1 observation

Reply:
So after much deliberation if the user (api or web) sends a request with incorrect dimensions, this will throw a 400 response with a list of incorrect dimensions.